about.html

```diff
@@ -0,0 +1,243 @@
+      It directs attention to new markets and fields of action for companies and regions, enables the joint exploration of visions and strategies, 
+      and extends the boundaries of previous knowledge. 
+    </p>
+    <p class="text-center mobileDescription">
+      The Kenya Women’s summit covers a wide range of aspects, 
+      from the vital questions of the world economy to initiatives 
+      which demonstrate exemplary actions in times of crisis and also helps spread the information all over the country.  
+    </p>
+    <div class="mt-5 d-flex justify-content-center 
+    align-items-center womenKenyaLogoDiv">
+    <img src="./images/womenlogo3.png" alt="logo">
+  </div>
+  </div>
+  <div class="container programLogo d-flex"></div>
+</article>
+<article class="pastEvents">
+  <h2 class="text-center seconday-dark">Previous Events</h2>
+  <div class="line"></div>
+  <div class="eventDescripition">
+    <p class="text-center my-3">
+      have a look at our accomplishments from previous events and the change brought by them
+    </p>
+  </div>
+  <div class="container events d-flex">
+    <div class="event event-left d-flex justify-content-center
+    align-items-center p-5">
+    <div>
+      <h4 class="text-white eventYear">2020</h4>
+      <p class="text-white">
+        Kenya Women’s Summit 
+      </p>
+    </div>
+  </div>
+  <div class="event-right event d-flex justify-content-center
+  align-items-center p-5">
+   <div>
+    <h4 class="text-white eventYear">2021</h4>
+    <p class="text-white">
+      Kenya Women’s Summit
+    </p>
+   </div>
+ </div>
+</div>
+</article>
+<article class="partners">
+  <h2 class="text-center secondary-light">partners</h2>
+  <div class="line"></div>
+  <div class="container d-flex partnerslogos">
+      <div class="partnerLogodiv">
+          <img src="./images/partner/google.svg" alt="google">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/mozilla.svg" alt="mozilla">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/microsoft.svg" alt="microsoft">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/zoom-communications.svg" alt="zoom">
+      </div>
+      <div class="partnerLogodiv">
+          <img src="./images/partner/airbnb.svg" alt="airbnb">
+      </div>
+  </div>
+</article>
+<footer>
+  <div class="container footerWrapper">
+      <div class="mt-5 footerLogoImage">
+        <a href="index.html"
+          ><img
+            src="./images/womenlogo3.png"
+            alt="women's logo"
+            class="footerWomenSummit"
+        /></a>
+      </div>
+      <div class="footerInfo">
+        <h4>@2022 Women's Summit.</h4>
+        <p class="mt-3">
+          Lorem ipsum dolor sit amet consectetur, adipisicing elit. Omnis,
+          suscipit veritatis!?
+        </p>
+      </div>
+    </div>
+  </footer>
```

Comment:
- [ ] Kindly change the background color for the footer section on the about page, to match the original design.

Your design
![image](https://user-images.githubusercontent.com/35535130/168910517-264602bb-80f1-4264-bd46-7a0bf13d3105.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/168910466-7c14c284-7849-415e-8d63-582634910b57.png)